docs/csharp/tour-of-csharp/types-and-variables.md

```diff
@@ -78,9 +78,9 @@ C#’s type system is unified such that a value of any type can be treated as an
 
 [!code-csharp[Boxing](../../../samples/snippets/csharp/tour/types-and-variables/Program.cs#L1-L10)]
 
-When a value of a value type is converted to type `object`, an `object` instance, also called a "box", is allocated to hold the value, and the value is copied into that box. Conversely, when an `object` reference is cast to a value type, a check is made that the referenced `object` is a box of the correct value type, and, if the check succeeds, the value in the box is copied out.
+When a value of a value type is assigned to an `object` reference, an `object` instance, also called a "box", is allocated to hold the value, and the value is copied into that box. Conversely, when an `object` reference is cast to a value type, a check is made that the referenced `object` is a box of the correct value type. If the check succeeds, the value in the box is copied to the value type.
```

Comment:
Here's a stab at avoiding the construct "an `object` reference, an `object` instance," which is a little hard to follow at first.
```suggestion
When a value of a value type is assigned to an `object` reference, an `object` instance is allocated to hold the value . The `object` instance is called a box, and the value is copied into that box. Conversely, when an `object` reference is cast to a value type, a check is made that the referenced `object` is a box of the correct value type. If the check succeeds, the value in the box is copied to the value type.
```

Reply:
I like that, and then modified the wording a bit more.